scania-digital-design-system/sdds-website
# feat(icons): gallery for SVG icons

**Describe pull-request**  
Adding new tab called "Icons" for display of SVGs

**Solving issue**  
Fixes: AB#1549

**How to test**  
1. Run Aplifly link from down below
2. Open Foundation -> Icons -> Tab Icon
3. Check downloading single icon (it should be svg file)
4. Check downloading multiple icons (it should be zip file)
5. Check downloading all icons (it should be zip file)
6. Check search
7. Check downloading from modal (open single icon)
8. Check text on-page for typos/wrong grammar etc.
